deployments/resolver.go

```diff
@@ -336,7 +336,7 @@ func resolveValueAssignment(kv *api.KV, deploymentID, nodeName, instanceName, re
 	}
 	if valueAssignment == nil {
 		ctx := events.NewContext(context.Background(), events.LogOptionalFields{events.NodeID: nodeName, events.InstanceID: instanceName})
-		events.WithContextOptionalFields(ctx).NewLogEntry(events.LogLevelWARN, deploymentID).Registerf("[WARNING] The value assignment %q hasn't be resolved for deployment: %q, node: %q, instance: %q. An empty string is returned instead", valueAssignment, deploymentID, nodeName, instanceName)
+		events.WithContextOptionalFields(ctx).NewLogEntry(events.LogLevelWARN, deploymentID).Registerf("[WARNING] The value assignment %q hasn't be resolved for deployment: %q, node: %q, instance: %q. An empty string is returned instead", va, deploymentID, nodeName, instanceName)
```

Comment:
the passed value assignment is parsed into `va` in line [328](https://github.com/ystia/yorc/pull/527/files#diff-434b86138c68a6e888c5e9568619124fR328)